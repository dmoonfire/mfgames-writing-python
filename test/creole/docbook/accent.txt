= Testing Accents

Acute: á é í ó ú

Grave: à è ì ò ù

Macron: ā ē ī ō ū

Circumflex: â ê î ô û
