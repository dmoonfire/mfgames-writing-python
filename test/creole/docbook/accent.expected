<?xml version="1.0" encoding="UTF-8"?><article xmlns='http://docbook.org/ns/docbook' version="5.0"><info><title>Testing Accents</title></info><simpara>Acute: á é í ó ú</simpara><simpara>Grave: à è ì ò ù</simpara><simpara>Macron: ā ē ī ō ū</simpara><simpara>Circumflex: â ê î ô û</simpara></article>
